### DISCUSSION TITLE
OC Profile for RTX3080

### DISCUSSION THREAD

**Discussion by astronyu** [RainbowMiner/RainbowMiner#1694] [Q&A] [answered]
Hi @RainbowMiner, can you suggest an ocprofile for RTX3080 as a base starting point?

**Comment by jeremyhammon** [comment]
NiceHash provides good reference values for many cards:

https://www.nicehash.com/blog/post/nvidia-and-amd-graphics-card-oc-settings-for-mining

**Comment by RainbowMiner** [comment]
Hi,
the Nicehash list is good, even though I think, some of the OC values are a bit high. Especially, if you have set "Force P2 State" to "Off". I have only GTX1070, RTX2070 and RTX3070 in my rigs, no 3080, but maybe you can use my RTX3070 profiles as starting point. Regarding Ethash mining, I had best results with the "Profile2". My RTX3070 mine away with 61.5MH/s with Gminer, 
 Ethminer, Trex or Phoenix. Gminer and Ethminer have shown higher effective pool hashrate, compared to Trex and Phoenix, currently.

These are the default profiles for my RTX3070
```
    "Profile1-RTX3070": {
      "PowerLimit": 80,
      "ThermalLimit": 0,
      "MemoryClockBoost": "0",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile2-RTX3070": {
      "PowerLimit": 80,
      "ThermalLimit": 0,
      "MemoryClockBoost": "1000",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile3-RTX3070": {
      "PowerLimit": 80,
      "ThermalLimit": 0,
      "MemoryClockBoost": "500",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile4-RTX3070": {
      "PowerLimit": 80,
      "ThermalLimit": 0,
      "MemoryClockBoost": "-1000",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile5-RTX3070": {
      "PowerLimit": 80,
      "ThermalLimit": 0,
      "MemoryClockBoost": "875",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
```

**Comment by astronyu** [comment]
> Hi,
> the Nicehash list is good, even though I think, some of the OC values are a bit high. Especially, if you have set "Force P2 State" to "Off". I have only GTX1070, RTX2070 and RTX3070 in my rigs, no 3080, but maybe you can use my RTX3070 profiles as starting point. Regarding Ethash mining, I had best results with the "Profile2". My RTX3070 mine away with 61.5MH/s with Gminer,
> Ethminer, Trex or Phoenix. Gminer and Ethminer have shown higher effective pool hashrate, compared to Trex and Phoenix, currently.
> 
> These are the default profiles for my RTX3070
> 
> ```
>     "Profile1-RTX3070": {
>       "PowerLimit": 80,
>       "ThermalLimit": 0,
>       "MemoryClockBoost": "0",
>       "CoreClockBoost": "100",
>       "LockVoltagePoint": "*",
>       "PreCmd": "",
>       "PreCmdArguments": "",
>       "PostCmd": "",
>       "PostCmdArguments": ""
>     },
>     "Profile2-RTX3070": {
>       "PowerLimit": 80,
>       "ThermalLimit": 0,
>       "MemoryClockBoost": "1000",
>       "CoreClockBoost": "100",
>       "LockVoltagePoint": "",
>       "PreCmd": "",
>       "PreCmdArguments": "",
>       "PostCmd": "",
>       "PostCmdArguments": ""
>     },
>     "Profile3-RTX3070": {
>       "PowerLimit": 80,
>       "ThermalLimit": 0,
>       "MemoryClockBoost": "500",
>       "CoreClockBoost": "100",
>       "LockVoltagePoint": "*",
>       "PreCmd": "",
>       "PreCmdArguments": "",
>       "PostCmd": "",
>       "PostCmdArguments": ""
>     },
>     "Profile4-RTX3070": {
>       "PowerLimit": 80,
>       "ThermalLimit": 0,
>       "MemoryClockBoost": "-1000",
>       "CoreClockBoost": "100",
>       "LockVoltagePoint": "*",
>       "PreCmd": "",
>       "PreCmdArguments": "",
>       "PostCmd": "",
>       "PostCmdArguments": ""
>     },
>     "Profile5-RTX3070": {
>       "PowerLimit": 80,
>       "ThermalLimit": 0,
>       "MemoryClockBoost": "875",
>       "CoreClockBoost": "100",
>       "LockVoltagePoint": "*",
>       "PreCmd": "",
>       "PreCmdArguments": "",
>       "PostCmd": "",
>       "PostCmdArguments": ""
>     },
> ```

Thank you, this is just what I need. I can then tweak it for 3080.

**Comment by astronyu** [comment, marked as answer]
Just want to share, after testing different oc values, these are the one stable oc that gives me good hashrates.

iGame RTX3080 Adv OC (OC button ON)
Eth 94.23 MH/s
244W

> {
    "Profile1-RTX3080": {
      "PowerLimit": 75,
      "ThermalLimit": 0,
      "MemoryClockBoost": "0",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile2-RTX3080": {
      "PowerLimit": 75,
      "ThermalLimit": 0,
      "MemoryClockBoost": "1050",
      "CoreClockBoost": "-150",
      "LockVoltagePoint": "",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile3-RTX3080": {
      "PowerLimit": 75,
      "ThermalLimit": 0,
      "MemoryClockBoost": "400",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile4-RTX3080": {
      "PowerLimit": 75,
      "ThermalLimit": 0,
      "MemoryClockBoost": "-900",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
    "Profile5-RTX3080": {
      "PowerLimit": 75,
      "ThermalLimit": 0,
      "MemoryClockBoost": "775",
      "CoreClockBoost": "100",
      "LockVoltagePoint": "*",
      "PreCmd": "",
      "PreCmdArguments": "",
      "PostCmd": "",
      "PostCmdArguments": ""
    },
}

**Comment by RainbowMiner** [comment]
Cool! Thank you for sharing. I will add some defaults to the config.
